portfolio/src/main/java/com/google/sps/servlets/date.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps;
+
+public final class date{
```

Comment:
Can you rename both the class and the file Date, with a capital D? Java class names are typically in upper camel case format. Style guide here: https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names